src/locales/fi.json

```diff
@@ -338,7 +339,9 @@
       "hullMaterial": "Runkomateriaali",
       "purpose": "Käyttötarkoitus",
       "inspection": "Katsastustodistus",
-      "insurance": "Vakuutuskirjaote"
+      "insurance": "Vakuutuskirjaote",
```

Comment:
Please consider removing this if it's no longer used.

Reply:
It's still in use.